src/types/registration.types.ts

```diff
@@ -83,6 +83,12 @@ interface AdditionalIdentifier {
   value: string;
 }
 
+interface PublicationNote {
+  type: 'UnpublishingNote' | 'publicationNote';
+  note?: string;
+  publicationNoteMessage?: string;
```

Comment:
hva er forskjellen på disse? hva skal brukes når

Reply:
Det er teit, jeg skal ta det med torbjørn, men UnpublishingNote har note, og PublicationNote har publicationMessage.

Reply:
_om_ de skal være ulike lønner det seg å ha 2 ulike interfaces da, eller er det 50% for at neste person som skal bruke dette i frontend bruker feil kombo av `type` og property-name. men om det skal slås sammen kan det bare stå, så må vi huske å fjerne den ene når vi gjør det i APIet